### PR TITLE
Ra/display/editor/templates

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/display-templates?view=aspnetcore-2.1&branch=pr-en-us-25959)

## Public review  of the doc build:
- download the [HTML files](https://www.dropbox.com/sh/yy994551cje0kap/AAB0tRnm7gze2kYrSYPSFHIda?dl=0)
- unzip
- view HTML file in a browser. It best to view the doc with the widest window that has no left or right TOC (Table  of contents)
- For links outside the doc, remove the `review.` from URL.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@Ducki please review
## Public review  of the doc build:
- download the [HTML files](https://www.dropbox.com/sh/yy994551cje0kap/AAB0tRnm7gze2kYrSYPSFHIda?dl=0)
- unzip
- view HTML file in a browser. It best to view the doc with the widest window that has no left or right TOC (Table  of contents)
- For links outside the doc, remove the `review.` from URL.

File: aspnetcore/mvc/views/display-templates.md [hunk 1]
```diff
@@ -9,59 +9,74 @@ no-loc: ["DisplayTemplate", "EditorTemplate", Home, Model, "Page Model", "Razor
...
+
+[Tag Helpers](xref:mvc/views/tag-helpers/intro) provide an alternative way enable server-side code to participate in creating and rendering HTML elements in Razor files. For more information, see [Tag Helpers compared to HTML Helpers](xref:mvc/views/tag-helpers/intro#tag-helpers-compared-to-html-helpers).
+
 ## Display templates
 
-*DisplayTemplates* are a way to customize the display of model fields or create a layer of abstraction between your model values and their display.
+*DisplayTemplates* customize the display of model fields or create a layer of abstraction between the model values and their display.
 
-A *DisplayTemplate* is a [Razor](xref:mvc/views/razor) markup file (`.cshtml`) file placed in a folder called `DisplayTemplates` in the *Shared* folder of your *Pages* or *Views* folder (depending on whether you are using Razor Pages or MVC).
+A *DisplayTemplate* is a [Razor](xref:mvc/views/razor) file placed in the`DisplayTemplates`folder:
```
Comment (Ducki) [inline aspnetcore/mvc/views/display-templates.md:36]: Space missing between _the`DisplayTemplates`_

File: aspnetcore/mvc/views/display-templates/sample/Pages/Shared/EditorTemplates/Address.cshtml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿@model Address
+
+
+<dl>
+    <dd>    Name:
```
Comment (Ducki) [inline aspnetcore/mvc/views/display-templates/sample/Pages/Shared/EditorTemplates/Address.cshtml:5]: redundant spacings before labels

File: aspnetcore/mvc/views/display-templates.md [hunk 3]
```diff
@@ -9,59 +9,74 @@ no-loc: ["DisplayTemplate", "EditorTemplate", Home, Model, "Page Model", "Razor
...
-You can also use templates for form controls. They are called *EditorTemplates* and work the same way as *DisplayTemplates*. The only difference is the name of the folder: instead of *DisplayTemplates*, you need to call it *EditorTemplates*.
 
-To reference them, use the <xref:Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperEditorExtensions.EditorFor*> HTML helper, for example:
-[!code-cshtml[](display-templates/sample/Pages/Index.cshtml?name=snippet_htmlEditorFor)]
+Editor templates are used in form controls when the model is edited or updated.
+
+A *EditorTemplate* is a [Razor](xref:mvc/views/razor) file placed in the`EditorTemplates`folder:
+
+* For Razor Pages apps, in the `Pages/Shared/EditorTemplates` folder.
+* For MVC apps, in the `Views//Shared/EditorTemplates` folder or the `Views/ControllerName/EditorTemplates` folder.
```
Comment (Ducki) [inline aspnetcore/mvc/views/display-templates.md:68]: Double slash in _`Views//Shared/EditorTemplates`_

**Review by Ducki — CHANGES_REQUESTED** [review]
Thank you for the revision! I only discovered some typo issues (see review comments).

Maybe I would have left a little mention (or TIP box), that the DisplayTemplates work with build-in types (like DateTime) as well?

**Pull request merged**